packages/build-utils/src/file-ref.ts

```diff
@@ -45,22 +45,55 @@ export default class FileRef implements FileBase {
     this.mutable = mutable;
   }
 
+  /**
+   * Retrieves the URL of the CloudFront distribution for the S3
+   * bucket represented by {@link getNowFilesS3Url}.
+   *
+   * @returns The URL of the CloudFront distribution
+   */
+  private getNowFilesCloudfrontUrl(): string {
+    return (
+      process.env.NOW_FILES_CLOUDFRONT_URL ||
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏Can we add some minimal URL validation here (and in the other methods) to make errors clearer if someone puts non-URLs in this env var?

Something like:

```
return getEnvUrl('NOW_FILES_CLOUDFRONT_URL') || 'https://dmmcy0pwk6bqi.cloudfront.net';
```

with:

```
function getEnvUrl(envVarKey: string): string {
  const value = process.env[envVarKey];
  if (!validateUrl(value)) {
    throw new Error(`Environment Variable "${envVarKey}" was set to a value that was not a valid URL: ${value}`);
  }
  return value;
}
```

with some reasonable implementation of a `validateUrl`

Reply:
Done in [`dde90f1`](https://github.com/vercel/vercel/pull/12643/commits/dde90f1f1ac854c9f08728d6b0af07b979fba396), but I've opted to not surface the value as part of the error message, in case it holds a sensitive value?